platform-sdk/swirlds-unit-tests/core/swirlds-platform-test/src/test/java/com/swirlds/platform/test/consensus/ConsensusTests.java

```diff
@@ -131,6 +142,9 @@ void forkingTests(final ConsensusTestParams params) {
                 .setParams(modifyParams(params))
                 .setIterations(NUM_ITER)
                 .run();
+        // Some forking tests make too many forkers.  When there is  > 1/3 nodes forking, no super majority can result.
```

Comment:
According to Leemon's slack message, it seems it would also be possible to have no judges at all. So theoretically, that marker file too should be ignored here (though in practice, it seems unlikely that that would arise)

Reply:
Adding